src/act.drive.cpp

```diff
@@ -2351,3 +2352,17 @@ int calculate_vehicle_entry_load(struct veh_data *veh) {
 ACMD(stop_rigging_first) {
   send_to_char(ch, "You'll need to stop rigging by using the %s command before you can do that.\r\n", AFF_FLAGGED(ch, AFF_RIG) ? "^WRIG^n" : "^WRETURN^n");
 }
+
+process_vehicle_decay() {
+  struct room_data *room;
+  struct veh_data *veh = NULL;
+    if (VEH_FLAGGED(*veh, VFLAG_LOOTWRECK) && !((veh->in_veh) || ROOM_FLAGGED(veh->in_room, ROOM_GARAGE)) ) {
+      if (veh->spare <= time(0)) {
+        snprintf(buf, sizeof(buf), "A hulking utility forklift drives up, lifts the remains of %s up, and drives off to the junkyard.\r\n", veh);
```

Comment:
This makes it sound like the vehicle is retrievable if they go to the junkyard, but it's extracted instead. "Drives off into the sunset with it" maybe.